### PR TITLE
Upvote

### PR DESCRIPTION
**Pull request by JoshuaFields**
added upvotes, downvotes, pagination, and fixed some issues

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -12,6 +12,22 @@ def create
...
+  def downvote
+    # @tutorial = Tutorial.find(params[:tutorial_id])
+    @review = Review.find(params[:id])
+    @review.downvote_by current_user
+    @tutorial = @review.tutorial
+    redirect_to tutorial_path(@tutorial)
+  end
+
+  def upvote
+    # @tutorial = Tutorial.find(params[:tutorial_id])
```
Comment (cmkoller) [inline app/controllers/reviews_controller.rb]: Delete these two commented-out lines if you aren't using them!

File: app/controllers/tutorials_controller.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
 class TutorialsController < ApplicationController
   def index
-    @tutorials = Tutorial.all
+    @tutorials = Tutorial.all.page(params[:page]).per(10)
   end
 
   def show
     @tutorial = Tutorial.find(params[:id])
     @review = Review.new
+    # @vote = Vote.new
```
Comment (cmkoller) [inline app/controllers/tutorials_controller.rb]: Delete commented out lines!

File: app/views/tutorials/index.html.erb [hunk 3]
```diff
@@ -1,7 +1,9 @@
+<%= link_to "Add Tutorial", new_tutorial_path, class: "button radius" %>
+
 <% @tutorials.each do |tutorial| %>
 	<%= link_to tutorial do %>
-		<h2><%= tutorial.title %></h2>
+		<h3><%= tutorial.title %></h3>
```
Comment (cmkoller) [inline app/views/tutorials/index.html.erb]: Fix the indentation on this section!

**Comment by cmkoller** [comment]
Before I can merge this, I need to see some feature tests on the voting functionality!

File: app/controllers/reviews_controller.rb [hunk 4]
```diff
@@ -12,6 +12,20 @@ def create
...
+    redirect_to tutorial_path(@tutorial)
+  end
+
+  def upvote
+    @review = Review.find(params[:id])
+    @review.upvote_by current_user
+    @tutorial = @review.tutorial
+    redirect_to tutorial_path(@tutorial)
+  end
+
```
Comment (radavis) [inline app/controllers/reviews_controller.rb]: While this is functional, this is deviating from the way we do things in Rails. We should be hitting the VotesController to create a new vote.

**Pull request merged**